web/viewer.html

```diff
@@ -27,6 +27,8 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+    <!-- Ignore legacy IE compatibility settings and prefer HTML5 -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Still have to validate with Edge, the other versions of Internet Explorer.

Will commit this change when testing is complete.